zuul.d/base.yaml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
We can drop this file as we will be running molecule jobs via prow.

Reply:
ack